RELEASE_NOTES.md

```diff
@@ -1,5 +1,17 @@
 # QuantumLeap Release Notes
 
+## 0.9.0-dev
```

Comment:
Guessing it's all still a WIP?

Reply:
it's the pr to kick of new dev cycle, so that's why is -dev